### PR TITLE
Release

### PR DESCRIPTION
**Pull request by adamdecaf**

### CODE REVIEW

**Comment by adamdecaf** [comment]
I want to get all the PR's I made merged, then we can release 1.0.0? Thoughts @rubbish ?

File: README.markdown [hunk 1]
```diff
@@ -4,6 +4,18 @@
 
 jsonz is a library that implements json serialization and deserialization via typeclasses in scala.
 
+# Getting
```
Comment (trentio) [inline README.markdown]: I would say more than just "Getting" here

**Comment by trentio** [comment]
:+1:

**Comment by trentio** [comment]
:100:

**Comment by rubbish** [comment]
This looks good to me. We could leverage the release plugin if you wanted to @adamdecaf

**Comment by adamdecaf** [comment]
@rubbish which release plugin?

**Comment by rubbish** [comment]
@adamdecaf the [sbt-release-plugin](https://github.com/sbt/sbt-release)

**Comment by adamdecaf** [comment]
@rubbish yea, we could use that. I've just done it manually before.

**Comment by zmccoy** [comment]
Do we want to use the release plugin then instead and get that going?

**Comment by adamdecaf** [comment]
@zmccoy I haven't gotten back to trying that. I more so wanted to get a 1.0 release to bintray.

**Comment by zmccoy** [comment]
All good.  I just wanted to make sure we didn't forget about it is all.  just did a pr sweep of things from ~2 weeks back and didn't want anything to starve.

**Pull request merged**